src/org/pursuit/Account.java

```diff
@@ -3,22 +3,19 @@
 import java.util.ArrayList;
 
 public class Account {
-
-
     private int accountNumber;
     private String accountType;
     private double accountBalance;
-    private Customer customer;
```

Comment:
I think a Customer has an account...Not so much that an Account has a Customer. so removing this field:     private Customer customer;
    I would imagine in some other class, we can say in like psvm() --> give me all accounts for customerA. Not that I find an account,
     and then query that account for which customer field it has...